documentation/commands/git-new-pull-request.md

```diff
@@ -12,7 +12,8 @@ git new-pull-request
 
 #### DESCRIPTION
 
-Opens a browser window to the form for a new pull request page
+Syncs the current branch,
+and opens a browser window to the form for a new pull request page
```

Comment:
Great suggestion! I went with

```
Syncs the current branch
and opens a browser window to the new pull request page of your repository.

The form is pre-populated for the current branch
so that the pull request only shows the changes made
against the immediate parent branch.